facilitators.html

```diff
@@ -135,7 +135,113 @@ <h2 class="new-color-text h2 fs-1 mb-4">How to Set Up Edsight in Four Steps</h2>
             </div>
         </div>
     </section>
-
+    <!-- Footer-->
+    <footer class="bg-black text-center py-5">
+        <div class="container px-5">
+            <div class="text-white-50 small">
+                <div class="mb-2">&copy; Your Website 2022. All Rights Reserved.</div>
```

Comment:
Instead of "Your Website 2022", can you change this to "UCI Design and Partnership Lab, 2022" ?